backend/testing/db/data/data.sql

```diff
@@ -0,0 +1,23 @@
+COPY bgp_updates (key, prefix, origin_as, peer_asn, as_path, service, type, communities, "timestamp", hijack_key, handled, matched_prefix, orig_path) FROM stdin;
```

Comment:
Nice that we can also do that! How are these pre-installed updates and hijacks used in testing?

Reply:
A so that you have data to check the frontend against, I suppose.